deploy/10_2_deploy_gold_gbp_liquidity_pools.ts

```diff
@@ -0,0 +1,31 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import { getWeth, getUniswapFactory, getBxau, getBgbp } from "../utils/DeployedContractsHelpers";
+import { deployPairOracle } from "../utils/DeploymentHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log("Starting deployment: liquidity pools");
+
+  const factory = await getUniswapFactory(hre);
+  const wethAddress = (await getWeth(hre)).address;
+
+  const bxau = await getBxau(hre);
+  const bgbp = await getBgbp(hre);
+  const stables = [bxau, bgbp];
+
+  for (const bdStable of stables) {
```

Comment:
@tkeidar we're missing in this file also calling the `feeTo` function.
When we create the initial pools we didn't do it, but since then we did so new stables need it.

Let's discuss if it should be the operational treasury or the treasury offline.

Reply:
feeTo is defined only once on the Factory contract

Reply:
You're right, my bad.